doc/introduction/measurements.rst

```diff
@@ -524,6 +524,80 @@ Collecting statistics for sequences of mid-circuit measurements is supported wit
     When collecting statistics for a list of mid-circuit measurements, values manipulated using
     arithmetic operators should not be used as this behaviour is not supported.
 
+Configuring mid-circuit measurements
+************************************
+
+As seen above, there are multiple ways in which circuits with mid-circuit measurements can be executed with
+PennyLane. For ease of use, we provide the following configuration options to users when initializing a
+:class:`~pennylane.QNode`:
+
+* ``mcm_method``: To set the method used for applying mid-circuit measurements. Use ``mcm_method="deferred"``
+  to use the deferred measurements principle or ``mcm_method="one-shot"`` to use the one-shot transform as
+  described above. When executing with finite shots, ``mcm_method="one-shot"`` will be the default, and
```

Comment:
Note that with TT being merged soon this default is changing I think? (#5180)

Reply:
Thanks for pointing that out. I'd like to leave this as is for now and then update it after TT is merged, or have this be updated in #5180 .